ipatests/test_integration/test_installation.py

```diff
@@ -163,7 +163,6 @@ def install(cls, mh):
     def test_replica1_ipa_kra_install(self):
         super(TestInstallWithCA1, self).test_replica1_ipa_kra_install()
 
-    @pytest.mark.xfail(reason="FreeIPA ticket 7008", strict=True)
```

Comment:
Is this related to this patch?

Reply:
That should have been in the temp commit. I'll remove it together with the temp commit.